site2/website/versioned_docs/version-2.3.1/io-redis.md

```diff
@@ -0,0 +1,28 @@
+---
+id: version-2.3.1-io-redis
```

Comment:
was redis connector included in 2.3.1? if it doesn't, we just need it documented in master.

Reply:
Not included yet, changed.